File: edabit/day-48/reFormWord.py

# https://edabit.com/challenge/vxKcxazrqgmNA64db

def get_word(left, right):
    return left.title() + right


a = "daniel hemmmati"
# print(a.title())
# print(get_word('daniel', 'hemmati'))

print(a.capitalize())



# other solution
# this one is better
def get_word(left, right):
	return (left+right).capitalize()

File: edabit/day-48/reFormWord.js

//https://edabit.com/challenge/KQQ7RMXjSK257GQgy


/**
 * 
 * @param {String} left 
 * @param {String} right 
 */
function getWord(left, right) {
  return left.charAt(0).toUpperCase() + left.slice(1)  + right
}

var a = "daniel hemmati"
// let res = a.charAt(0).toUpperCase() + a.slice(1)

// console.log(res)

console.log(getWord(a))

